tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -46,42 +43,43 @@ describe(__filename, () => {
 
   function render({ location, ...props } = {}) {
     const errorHandler = createStubErrorHandler();
-
-    return shallowUntilTarget(
-      <AddonMoreInfo
-        addon={props.addon || createInternalAddonWithLang(fakeAddon)}
-        errorHandler={errorHandler}
-        i18n={fakeI18n()}
-        store={store}
-        {...props}
-      />,
-      AddonMoreInfoBase,
-      {
-        shallowOptions: createContextWithFakeRouter({ location }),
-      },
-    );
+    const allProps = {
+      addon: createInternalAddonWithLang(fakeAddon),
+      errorHandler,
+      ...props,
+    };
+
+    const renderOptions = { store };
+    if (location) {
+      renderOptions.history = createMemoryHistory();
+      renderOptions.history.push(location);
+    }
+
+    return defaultRender(<AddonMoreInfo {...allProps} />, renderOptions);
   }
 
   it('renders LoadingText if no add-on is present', () => {
-    const root = render({ addon: null });
+    const { queryByClassName } = render({ addon: null });
 
     // These fields will be visible during loading since
     // they will always exist for the loaded add-on.
-    expect(root.find('.AddonMoreInfo-last-updated')).toHaveLength(1);
+    expect(screen.getByText('Last updated')).toBeInTheDocument();
 
-    expect(root.find(LoadingText)).toHaveLength(1);
+    expect(queryByClassName('LoadingText')).toHaveLength(1);
```

Comment:
We have to use `queryByClassName` because there is nothing added to the DOM for `LoadingText` except a `<span>` with a class of `LoadingText`. We have to get access to `queryByClassName` by destructuring `render` because it is a custom query that I created and therefore is not added to the `screen` object. I am looking into whether there is a way to add our custom queries to the `screen` object, but I haven't found a way yet.

Reply:
I guess we could have our own `screen` object that's a wrapper around react testing libraries `screen` object?  (Or was that tried and didn't work)

Reply:
I got this to work and it's included in the next commit.